### PR TITLE
[test] Add test for issue#5980

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Add test for issue #5980
- This case is failed on failed version 0.23.5, pass on fixed version 0.27.4

### CODE REVIEW

**Comment by wanghongjuan** [comment]
@rogerwang Could you please review this test at first, since I updated nw_util.py and I have other tests are depend on this change?

**Comment by wanghongjuan** [comment]
I just retest it and find it is not necessary, remove it.

File: test/sanity/issue5980-aws-sdk-embedded-youtobe-video-crash/test.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+node_module = os.path.join(testdir, "node_modules")
+os.chdir(testdir)
+
+try:
+    shutil.rmtree(node_module)
+except:
+    pass
+
+assert(False == os.path.isdir(node_module))
+subprocess.call(["npm", "install"])
```
Comment (rogerwang) [inline test/sanity/issue5980-aws-sdk-embedded-youtobe-video-crash/test.py]: may need to use 'install_native_modules' in nw_utils here.

Comment (wanghongjuan) [reply test/sanity/issue5980-aws-sdk-embedded-youtobe-video-crash/test.py]: updated

**Pull request merged**